src/main/java/ch/njol/skript/expressions/ExprItemThrower.java

```diff
@@ -0,0 +1,91 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.Bukkit;
+import org.bukkit.OfflinePlayer;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.Item;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.UUID;
+
+@Name("Item Thrower")
+@Description({
+	"The entity that threw/dropped the dropped item.",
```

Comment:
isnt the one that dropped the item the owner of the item? this should include an example

Reply:
No, owner is something Minecraft actually doesn't use anywhere. (can be used via Bukkit or commands)
Owner would signify who can pickup the item. 
Thrower is who dropped it.

Reply:
^ @TheAbsolutionism the owner and thrower description should make this distinction clear

Reply:
iirc, when you throw/drop something, you dont become the owner of it automatically.
The owner is used to make it where, if owner is set, then only the owner can pick it up.